src/sentry/integrations/vercel/integration.py

```diff
@@ -37,6 +38,18 @@
     ),
 ]
 
+INSTALL_NOTICE_TEXT = (
+    "Visit the Vercel Marketplace to install this integration. After installing the"
+    " Sentry integration, you'll be redirected to adding projects."
```

Comment:
nit: the last part is a little confusing to me, maybe something like:
> "you'll be redirected back to Sentry to finish syncing Vercel and Sentry projects."

(that is what's happening right...?)